js/interface.js

```diff
@@ -2189,11 +2197,24 @@ $allowBtnFilter.click(function(event) {
 
   if (value === 'tokens') {
     var tpl = Fliplet.Widget.Templates['templates.apiTokenList'];
+    var appTokens = _.groupBy(integrationTokenList, function (token) {
+      return _.get(_.first(token.apps), 'name', DESCRIPTION_APP_UNKNOWN);
+    });
+
+    // Sort by key (app name), but keep the unknown grouped tokens at the end of the list
+    var appsList = _.sortBy(_.mapValues(appTokens, function (tokens, name) {
+      return { name: name, tokens: tokens };
+    }), function (key) {
+      return key === DESCRIPTION_APP_UNKNOWN ? _.repeat('z', 32) : key;
```

Comment:
```suggestion
      return key === DESCRIPTION_APP_UNKNOWN ? 'z' : key.toUppercase();
```

...so that the sort is case-insensitive.